<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[browser opened]]>
    </line>
    <line>
      <![CDATA[log in app]]>
    </line>
    <line>
      <![CDATA[TC executed]]>
    </line>
  </reporter-output>
  <suite name="Suite" duration-ms="34843" started-at="2020-09-24T23:19:25 IST" finished-at="2020-09-24T23:20:00 IST">
    <groups>
    </groups>
    <test name="Chrome" duration-ms="34843" started-at="2020-09-24T23:19:25 IST" finished-at="2020-09-24T23:20:00 IST">
      <class name="mudule.test.login.KiteLoginTest">
        <test-method status="PASS" signature="browserOpen(java.lang.String)[pri:0, instance:mudule.test.login.KiteLoginTest@71c3b41]" name="browserOpen" is-config="true" duration-ms="14158" started-at="2020-09-24T23:19:26 IST" finished-at="2020-09-24T23:19:40 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[browser opened]]>
            </line>
          </reporter-output>
        </test-method> <!-- browserOpen -->
        <test-method status="PASS" signature="loginKite()[pri:0, instance:mudule.test.login.KiteLoginTest@71c3b41]" name="loginKite" is-config="true" duration-ms="2411" started-at="2020-09-24T23:19:40 IST" finished-at="2020-09-24T23:19:42 IST">
          <reporter-output>
            <line>
              <![CDATA[log in app]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginKite -->
        <test-method status="FAIL" signature="verifyProfileName()[pri:0, instance:mudule.test.login.KiteLoginTest@71c3b41]" name="verifyProfileName" duration-ms="11878" started-at="2020-09-24T23:19:42 IST" finished-at="2020-09-24T23:19:54 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [KV1] but found [KV]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [KV1] but found [KV]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at mudule.test.login.KiteLoginTest.verifyProfileName(KiteLoginTest.java:55)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[TC executed]]>
            </line>
          </reporter-output>
        </test-method> <!-- verifyProfileName -->
        <test-method status="PASS" signature="logOutKite(org.testng.ITestResult)[pri:0, instance:mudule.test.login.KiteLoginTest@71c3b41]" name="logOutKite" is-config="true" duration-ms="4128" started-at="2020-09-24T23:19:54 IST" finished-at="2020-09-24T23:19:59 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyProfileName status=FAILURE method=KiteLoginTest.verifyProfileName()[pri:0, instance:mudule.test.login.KiteLoginTest@71c3b41] output=TC executed]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logOutKite -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:mudule.test.login.KiteLoginTest@71c3b41]" name="closeBrowser" is-config="true" duration-ms="1189" started-at="2020-09-24T23:19:59 IST" finished-at="2020-09-24T23:20:00 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- mudule.test.login.KiteLoginTest -->
    </test> <!-- Chrome -->
  </suite> <!-- Suite -->
</testng-results>
